### ISSUE TITLE
docker-compose up fails

### ISSUE THREAD

**Issue by shadyabhi** [Netflix/lemur-docker#39]
I've attached full error log with the issue. 

Environment:- 

```
➤ docker-compose --version
docker-compose version 1.23.2, build 1110ad01
➤ docker --version
Docker version 18.09.2, build 6247962
➤ sw_vers
ProductName:    Mac OS X
ProductVersion: 10.14.5
BuildVersion:   18F203
```
Error: 
```
Collecting black==19.3b0 (from lemur[tests])
  ERROR: Could not find a version that satisfies the requirement black==19.3b0 (from lemur[tests]) (from versions: none)
ERROR: No matching distribution found for black==19.3b0 (from lemur[tests])
Makefile:7: recipe for target 'develop' failed
make: *** [develop] Error 1
ERROR: Service 'lemur-web' failed to build: The command '/bin/sh -c git config --global url."https://".insteadOf git:// &&  cd /usr/local/src &&  git clone https://github.com/netflix/lemur.git &&  cd lemur &&  git checkout ${LEMUR_VERSION} &&  pip install --upgrade pip virtualenv &&  export PATH=/usr/local/src/lemur/venv/bin:${PATH} &&  virtualenv -p python3 venv &&  . venv/bin/activate &&  make ${LEMUR_TARGET}' returned a non-zero code: 2
```

Full error log: [error.log](https://github.com/Netflix/lemur-docker/files/3379876/error.log)

**Comment by abawchen** [comment]
same here :(

**Comment by drewby08** [comment]
Ran into the same issues.  This should fix it: https://github.com/Netflix/lemur-docker/pull/40

**Comment by shadyabhi** [comment]
Hey @drewby08 , 

Thanks for the link to PR. I was able to start it successfully via docker.

**Comment by kevgliss** [comment]
Fixed by #40
